config/locales/devise.ja.yml

```diff
@@ -0,0 +1,7 @@
+en:
```

Comment:
Delete this file
@iamdavidpetric what's up doc?

Reply:
done